.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,17 @@
+**Type of PR**: Bugfix / Feature / Docs / Other
+
+**Required reviews**: 1 / 2 / 3
+
+**What does this PR do? Why is it needed?**
```

Comment:
I suggest providing titles that remain relevant without needing to make any edits.  This makes templates less annoying to use.

```suggestion
**What this does**

> What does this PR do and Why is it needed?
```

Reply:
What do you mean by "needing to make any edits"? The title of that question is not meant to be edited (you can, of course, but it's not the intention).

Reply:
Ok sure. I mean that it's best if the PR does not need to make changes or edits to the template before opening the PR.  As an example, consider our current Issue template, bland needing to remove "optional".